source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,278 +3,577 @@
+   chroot-builder, this directory is where the bundle definition files are
+   temporarily stored while building chroots. By default, this directory is not
+   generated until it is needed, and is not generated at all if using the new
+   chroot-builder built into mixer. (More on this below in Step 8.)
+
+   The :file:`.yum-mix.conf` file defined in ``YUM_CONF`` is auto-generated,
+   along with the ``CERT`` file, :file:`Swupd_Root.pem`. The yum configuration
+   file is needed for the chroot-builder to know where the RPMs are hosted,
+   and the certificate file is needed to sign the root Manifest to provide
+   security for content verification.
+
+   You can change the ``CERT=/path/to/cert`` line to point to a different
+   certificate. The chroot builder inserts the certificate specified here
+   in ``/os-core-update/usr/share/clear/update-ca/``. This is the certificate
+   used by the software update client to verify the :file:`Manifest.MoM`
+   signature. For now, we *highly* recommend that you do not modify this
+   line, as the certificate that swupd expects needs a very specific
+   configuration to sign and verify properly. The certificate is
+   automatically generated, and the :file:`Manifest.MoM` is signed
+   automatically as well, providing security for the updated content that
+   you create.
+
+   The ``CONTENTURL`` and ``VERSIONURL`` should be set to the domain or IP
+   address where your updated content will be served. This is the location
+   that hosts the :file:`/home/clr/mix/update/www` (``SERVER_STATE_DIR``)
+   directory. Creating a symlink to the directory in your server webdir is
+   an easy way to host the content. These URLs are embeded in images created
+   for your mix. They are where ``swupd-client`` will look to figure out if
+   there is a new version available, and the location from which to download
+   the updated content. Think of these as the equivalent of the 
+   `ClearLinux update page`_ used by Clear Linux, but for your derivative mix.
+
+   To learn more about the ``FORMAT`` option, refer to the "Format Version"
+   section at the bottom of this document, and `Format Bumps`_ on the Clear
+   Linux wiki. For now, leave the ``FORMAT`` value alone and do not increment
+   it.
+
+   The mix version and upstream Clear version will come from two state files:
+   :file:`mixversion` and :file:`upstreamversion`, both of which will be
+   created for you when you set up the workspace. They will be created in the
+   directory defined by the ``VERSIONS_PATH``.
+
+#. **Create/locate RPMs for mix.**. (Steps 4 through 6 are necessary only
+   if you want to add your own RPMs to the Mix. If you are working only with
+   Clear bundles, then skip to Step 7.)
+
+   If you are creating RPMs from scratch, you can use ``autospec``, ``mock``,
+   ``rpmbuild``, etc. to build them. If they are not built on Clear,
+   make sure your configuration and toolchain builds them correctly for Clear,
```

Comment:
That is a good question.  I didn't change the wording here (it appears as new lines, but it's just been reformatted) because I don't know enough about Clear's RPM requirements to articulate it.

Could @matthewrsj @phmccarty or someone elaborate on what could go here?

Reply:
We can link to https://github.com/clearlinux/common for this section. I don't want to explain any of it in here though, because it will make it too long and off topic.

Reply:
@tmarcu I agree, the link is sufficient.